tests/ci/VTOOLS_VERSION

```diff
@@ -0,0 +1 @@
+79cdd85b11c55e66379a9829e28fe059391a14b6
```

Comment:
is there a particular reason why the file name is all caps?

Reply:
No particular reason. Probably a habit from capitalizing consts ? :smile: